### ISSUE TITLE
Lifecycle docs comments - 20160916

### ISSUE THREAD

**Issue by thisisaaronland** [whosonfirst/whosonfirst-cookbook#1]
https://github.com/whosonfirst/whosonfirst-cookbook/commit/e33288ca3508fd7a814fecbd47d5d719ef266a0e
## 

```
Cessation events (but when other feature(s) replace it with inception events)
```

This phrase reads a bit funny. It sort of reads "This thing. But not this thing. Unless it's thing with that thing." Maybe just talk about when a record is replaced or superseded, with a note that there is a discussion to follow.

```
Updates and edits that qualify as a Significant Event:
```

Off the top of my head one of the things that's missing would be one of the ancestors in a record's hierarchy changing. Think Danzig.

I would change the wording to indicate two things:
1. The following things are absolutely significant events
2. The list as it written today may be incomplete or unable to capture the subtleties and demands of real-life

The important point is, as you say later on, that WOF is "not in the business of removing features from history".

```
Changes to the geometry, where more than 50% (area or length) is added or removed
Moving a feature's location more than ten kilometers from it's original location
```

Why 50% Why 10km? Why not 5km? It doesn't need a treatise to explain why but a sentence or two discussing the rationale would be good.

```
When a Significant Event occurs, a new wof:id is minted for a new feature; this new feature replaces the old feature that required edits.
```

How does it replace it? I mean, I know what you're talking about but it's a bit confusing because it sounds like we are both never replacing things and replacing them all the same.

```
The supersedes and superseded_by values are updated in the respective records which allows a user to track the life of a given record in Who's on First.
```

Imagine I am reading this for the first time. Like I am a Busy Human Geographer who has a Google News Alert for "lifecycle" and this is the first time I've seen WOF. What are these supersedes/superseded_by properties?

Again, it can simply be a framing sentence rather than a digression in to the details.

```
Additionally, what Who's On First considers a Significant Event may be different ... their data usage.
```

This should be moved up closer to the introduction

```
...the history of any given feature at any given time
```

Again, maybe just be explicit that this ability is not inherent to the linked list but the linked list _and_ the Git (or more generic audit trail) log.

```
Lifecycle flowchart
```

It _is_ big! It's a good thing that I've been working on a generic image slippy-tile thing. We can use it for the blog post... https://github.com/thisisaaronland/go-iiif/blob/master/README.md

```
In USA? / Is the parent a new region
```

No.

We should absolutely not encode rules like that. They will only cause sadness. And make people angry. Also it's a bit arbitrary and doesn't square with the basic first principles of WOF.

Again, the point shouldn't be to identify things that are not a lifecycle event but that rather things that definitely _are_ lifecycle events.

Life is complicated. We will not be able to capture all the possible reasons that something changes but we can identify some of them.

```
Both of these categories can have one of two edits made: geometry edits or attribute edits (or both).
```

This sentence contradicts itself.

```
The "Life" section
```

This is just repeated stuff in the intro and flowchart.

```
Similar to the Yugoslavia example, above,
```

There is no Yugoslavia above. I know there used to be but it's not there now. This happens again later on in the document.

```
...would receive a date (the date of it's error correction) in the edtf:deprecated field
```

Busy Human Geographer wants to know what "edtf:deprecated" is

```
The Birth, Life, Death nomenclature
```

It starts to get a bit creepy, really. Also WOF strives for generic, to the point of bland, nomenclature for this sort of thing. I would find language that is a bit less loaded.

```
The new countries, Slovenia, Croatia, Bosnia and...
```

We should link to those places.
## 

cc @nvkelso @dphiffer

**Comment by stepps00** [comment]
Thanks @thisisaaronland! Updating shortly.

**Comment by stepps00** [comment]
@thisisaaronland - I took care of your comments here : https://github.com/whosonfirst/whosonfirst-cookbook/commit/f35a4c0d5e7c4c8ae90a0959fa14392f494be538. 

Notes based on your comments:

> _Off the top of my head one of the things that's missing would be one of the ancestors in a record's hierarchy changing._

Wouldn't this require us to mint new a `wof:id` for _any_ descendant of a feature that's been updated? For instance, when we changed neighbourhood records in Mexico City - this rule would require us to update all descendants with a new record, correct?

> There is no Yugoslavia above. I know there used to be but it's not there now. This happens again later on in the document.

Updated the wording around the Yugoslavia, but I'm going to keep this example in the document, as we discussed adding a Yugoslavia record to WOF. I can swap this out for a different example if we decide to take another route.

cc @nvkelso
